pallets/pallet-mock-skip-blocks/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "instant-seal")]
+pub use pallet::*;
+
+#[cfg(not(feature = "instant-seal"))]
+pub use dummy as pallet;
+
+#[cfg(not(feature = "instant-seal"))]
+#[frame_support::pallet]
+pub mod dummy {
+	use frame_support::pallet_prelude::*;
+
+    #[pallet::pallet]
+    pub struct Pallet<T>(_);
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {        
+        /// Dummy event
+       DummyEvent,
+	}
+
+    #[pallet::config]
+    pub trait Config: frame_system::Config {
+		/// Overarching event type
+        type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+	}
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {}
+}
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
```

Comment:
If the whole content of mock and test are only supposed to be used when `instant-seal` is available, wouldn't it make more sense to instead move the feature flag macro above the `mod` imports here? Then we can remove all the annotations from the `mock.rs` and `test.rs` files itself.

Reply:
True 👍🏼  I forgot about those macro flags when testing some things earlier

Reply:
Maybe we can even move them into the `pub mod pallet {}` definition? Not sure if that works but then no extra macros were needed.